pkg/plugin/packetparser/packetparser_bpfel_arm64.go

```diff
@@ -49,12 +55,18 @@ type packetparserPacket struct {
 		Tsval  uint32
 		Tsecr  uint32
 	}
-	ObservationPoint uint8
-	TrafficDirection uint8
-	Proto            uint8
-	Flags            uint8
-	IsReply          bool
-	_                [3]byte
+	ObservationPoint  uint8
+	TrafficDirection  uint8
+	Proto             uint8
+	Flags             uint8
+	IsReply           bool
+	_                 [3]byte
+	ConntrackMetadata struct {
+		BytesForwardCount   uint64
+		BytesReplyCount     uint64
+		PacketsForwardCount uint64
+		PacketsReplyCount   uint64
```

Comment:
This is probably right, but if we don't need to track connections with more than 4billion in both directions, then uint32 would suffice for the packet counts